lib/db/namespaces.js

```diff
@@ -0,0 +1,79 @@
+const { fetch } = require('../fetch');
+const FormData = require('form-data');
+const { get: getOptions } = require('./options');
+const lodashGet = require('lodash/get');
+
+const { httpBody } = require('../response-transforms');
+
+// For Stardog <= 7.0.2 compatibility. Remove later.
+const backwardsCompatGetNamespaces = (conn, database, params) =>
+  getOptions(conn, database, params).then(res => {
```

Comment:
should this only request `database.namespaces`? where you pass in an input json object with only that key.

Reply:
It can't do that right now, at least unless I rewrite (or don't use) `options.get`, because `options.get` always sends the full `DB_OPTIONS`. Since this was the default behavior anyway, and this is only here for backwards compatibility, and changing `options.get` seems orthogonal to this PR's goal, I'd like to leave it as-is for now.

Reply:
fair